tools/schemacode/bidsschematools/validate_schema.py

```diff
@@ -0,0 +1,17 @@
+import json
+from importlib.resources import files
+
+from jsonschema import ValidationError, validate
+
+
+def validate_schema(schema):
```

Comment:
I think it makes sense to include this in `bidsschematools.schema` rather than its own module.